packages/tag/src/Tag/Tag.tsx

```diff
@@ -227,10 +228,12 @@ const Default = <T extends TgphElement>({
     <Root color={color} size={size} variant={variant} {...props}>
       {icon && <Icon {...icon} />}
       <Text as="span">{children}</Text>
-      {onRemove && (
+      {onRemove && size !== "0" && (
         <Button onClick={onRemove} icon={{ icon: Lucide.X, alt: "Remove" }} />
       )}
-      {onCopy && <CopyButton onClick={onCopy} textToCopy={textToCopy} />}
+      {onCopy && size !== "0" && (
+        <CopyButton onClick={onCopy} textToCopy={textToCopy} />
+      )}
```

Comment:
I think gating this by `size` is an unnecessary abstraction for this.

Reply:
If we don't do that then we need to build a size 0 button fwiw

Reply:
Yeah, I get that it won't work. But nothing is stopping you from doing this either. 
```
<Tag.Root size="0">
   <Tag.Button>..</Tag.Button>
</Tag.Root>
```

Just don't want to muddy up components with inline logic like this.

Reply:
Ah got it. So how should we resolve though? Introduce a size 0 button?

Reply:
Yeah I think that makes the most sense, I'm sure we will run across a need for one at some point too.

Reply:
We can add that as a TODO item in Linear and merge this for now though